server/config/configuration.go

```diff
@@ -0,0 +1,22 @@
+package config
```

Comment:
I'm not a big fan of pushing this into a discrete package, especially when it's just one file right now.